### PR TITLE
ActivityReminder: Add setting to unlock screen

### PR DESCRIPTION
**Pull request by myxor**

### CODE REVIEW

File: apps/activityreminder/settings.js [hunk 1]
```diff
@@ -75,7 +75,14 @@
           settings.tempThreshold = v;
           activityreminder.writeSettings(settings);
         }
-      }
+      },
+      'Unlock on alarm': {
+        value: settings.unlock,
```
Comment (gfwilliams) [inline apps/activityreminder/settings.js:80]: You might want `value: !!settings.unlock,` here?

If `unlock` wasn't previously set (an old install?) this won't know it's meant to be a bool so won't be editable

File: apps/activityreminder/alert.js [hunk 2]
```diff
@@ -26,6 +26,13 @@
     if (!(storage.readJSON('setting.json', 1) || {}).quiet) {
       Bangle.buzz(400);
     }
+
+    if ((storage.readJSON('activityreminder.s.json', 1) || {}).unlock) {
+      console.log("unlock");
```
Comment (gfwilliams) [inline apps/activityreminder/alert.js:31]: was the print here intentional, or just for debugging?

**Comment by myxor** [comment]
I fixed your remarks @gfwilliams . Thanks!

**Comment by gfwilliams** [comment]
Thanks!

**Pull request merged**